File: 0884) Uncommon Words from Two Sentences.java
class Solution {
    public String[] uncommonFromSentences(String s1, String s2) {
        String s = s1 + " " + s2;
        Map<String, Integer> mp = new HashMap<>();
        String[] str = s.split(" ");
        for(String temp : str)
        {
            mp.put(temp, mp.getOrDefault(temp, 0) + 1);
        }
        List<String> ans = new ArrayList<>();
        for(Map.Entry<String, Integer> it : mp.entrySet())
        {
            if(it.getValue() == 1)
            {
                ans.add(it.getKey());
            }
        }
        return ans.toArray(new String[0]);
    }
}

File: 0884) Uncommon Words from Two Sentences.cpp
class Solution {
public:
    vector<string> uncommonFromSentences(string s1, string s2) {
        string s = s1 + " " + s2;
        unordered_map<string, int> mp;
        stringstream ss(s);
        string temp;
        while(ss >> temp)
        {
            mp[temp]++;
        }
        vector<string> ans;
        for(auto it : mp)
        {
            if(it.second == 1)
            {
                ans.push_back(it.first);
            }
        }
        return ans;
    }
};
